indico/htdocs/js/indico/modules/designer.js

```diff
@@ -470,20 +471,33 @@
             },
             clear_background: !template.background_url,
             title: $('.js-template-name').val(),
+            disassociate_backsides: showConfirmationDialog,
             backside_template_id: backsideTemplateID
         };
 
-        $.ajax({
-            url: location.pathname,
-            data: JSON.stringify(templateData),
-            contentType: 'application/json',
-            method: 'POST',
-            complete: IndicoUI.Dialogs.Util.progress(),
-            error: handleAjaxError,
-            success: function(data) {
-                handleFlashes(data, true);
-            }
-        });
+        if (showConfirmationDialog) {
```

Comment:
ok with the long call it's quite awful too. but i meant it more in general (creating the deferred and then using `then()` just once), doesn't necessarily need to use the ternary operator.